testing/jormungandr-integration-tests/src/common/jcli/command/transaction.rs

```diff
@@ -162,6 +162,7 @@ impl TransactionCommand {
         self
     }
 
+    #[allow(clippy::wrong_self_convention)]
     pub fn to_message<P: AsRef<Path>>(mut self, staging_file: P) -> Self {
```

Comment:
I guess this is justified because there is a command by that name. Or rename to something like `cmd_to_message`.